src/components/TemplateManagement.vue

```diff
@@ -29,48 +30,40 @@
 				@delete="onDeleteTemplate(t)"
 				@submit="onEditTemplate" />
 		</div>
-		<div v-if="!admin">
-			<h3>
+		<div v-if="!admin" class="link-to-admin-settings">
+			<h2>
 				{{ t('text_templates', 'Admin-defined templates') }}
-			</h3>
+			</h2>
+			<p> {{ t('text_templates', 'As an admin, you can create and manage text templates for all users.') }}</p>
```

Comment:
```suggestion
			<p> {{ t('text_templates', 'Admins can create and manage text templates for everyone.') }}</p>
```

And if we recognize the person is an admin, we could put a deeplink to the relevant admin settings there?

Reply:
That's what the button below does :)